### PR TITLE
Add ISSUE_TEMPLATES & configure templates

### PR DESCRIPTION
**Pull request by verkaufer**
This MR adds [issue templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository) to the repository.

The aim of this MR is to help us better manage bug reports and feature requests for the Node SDK.

### CODE REVIEW

File: .github/ISSUE_TEMPLATES/feature_request.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+### Summary
+<!-- Briefly describe the feature in one or two sentences. You can include more details later. -->
+
+### Use cases
+<!-- Describe the use cases that make this feature useful to others.
+The description should help the reader understand why the feature is necessary.
+The better we understand your use case, the better we can help create an appropriate solution. -->
+
+
+### How should it work?
```
Comment (jpcoenen) [inline .github/ISSUE_TEMPLATES/feature_request.yml:19]: Small suggestion:
```suggestion
### Proposed solution
```

File: .github/ISSUE_TEMPLATES/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+-->
+Connect Server Version:
+
+<!-- What OS was the SDK running on? -->
+OS:
+
+<!-- What version of Node and TypeScript were you running when the error occurred? -->
+Node Version:
+
+TypeScript Version:
```
Comment (jpcoenen) [inline .github/ISSUE_TEMPLATES/bug_report.yml:27]: How often would we need the TypeScript version? I am considering the option to just leave it out and ask for it if it is relevant for the submitted issue. This lowers the friction of submitting a bug (I wouldn't even know where to find this without Googling), while still addressing all relevant cases. What do you think?
```suggestion
<!-- What version of Node were you running when the error occurred? -->
Node Version:
```

**Review by jpcoenen — APPROVED** [review]
I have 2 minor suggestions. But apart from that, it looks good to go to me :rocket:

File: .github/ISSUE_TEMPLATES/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+---
+
+### Your environment
+
+<!-- Version of the SDK when the error occurred -->
+SDK Version:
+
+<!-- What version of the Connect server are you running?
+You can get this information from the Integrations section in 1Password
+https://1password.com/integrations/active
```
Comment (jpcoenen) [inline .github/ISSUE_TEMPLATES/bug_report.yml:17]:
```suggestion
https://start.1password.com/integrations/active
```

File: .github/ISSUE_TEMPLATES/feature_request.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+---
+name: Feature request
+about: Suggest an idea for the connector
+title: ''
+labels: feature-request
+assignees: ''
+
+---
```
Comment (jpcoenen) [inline .github/ISSUE_TEMPLATES/feature_request.yml:8]: The file needs a `.md` extension, same for the other template.

And the overall directory should be singular, i.e. `ISSUE_TEMPLATE`.

Comment (jpcoenen) [reply .github/ISSUE_TEMPLATES/feature_request.yml]: I've already fixed this.

File: .github/ISSUE_TEMPLATES/bug_report.yml (hunk 3 above)
Comment (jpcoenen) [reply .github/ISSUE_TEMPLATES/bug_report.yml]: Fixed.

File: .github/ISSUE_TEMPLATES/bug_report.yml (hunk 2 above)
Comment (verkaufer) [reply .github/ISSUE_TEMPLATES/bug_report.yml]: I added it proactively, but I think you're correct that we won't be too concerned about the TS version. 

I realize now that we distribute the package pre-compiled so it shouldn't be much of a concern unless the bug report comes from someone extending the SDK.

**Pull request merged**